src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -168,5 +168,40 @@ internal static DSAParameters ExportDsaParameters(SafeDsaHandle key, bool includ
             int yLength,
             byte[]? x,
             int xLength);
+
+        /// <summary>
+        /// Return the maximum value in the array; assumes non-negative values.
+        /// </summary>
+        private static int GetMax(int[] values)
```

Comment:
These don't seem particularly relevant to DSA...

Reply:
They’re used by the interop code for DSA and ECDSA. When I split the ECDSA code, I needed these methods to live somewhere bin the Android build since the Android ECDSA interop is in the AndroidCrypto class now.